### PR TITLE
Remove addons advertising in themes introduction.

### PR DESCRIPTION
**Pull request by jf-viguier**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 8.x
| Description?  | Remove addons publicity.
| Fixed ticket? | no

### CODE REVIEW

**Comment by kpodemski** [comment]
I'm not sure about this change. I would still recommend Addons Marketplace and maybe just add "and others" here and there.

WDYT @ttoine @eternoendless ?

**Comment by jf-viguier** [comment]
The dev doc is for the open source version or prestashop corp version ?
Open source version shouldn't promote addons more than another

File: themes/_index.md
Comment (Progi1984) [inline themes/_index.md]:
```suggestion
Several marketplaces are available to buy PrestaShop themes.
```

**Review by Progi1984 — COMMENTED** [review]
@jf-viguier Thanks. A small feedback

**Review by Progi1984 — APPROVED** [review]

**Comment by eternoendless** [comment]
I'm not sure if the documentation needs to be 100% neutral. One option would be to list "partner" marketplaces. After all, PrestaShop SA still largely funds PrestaShop development. WDYT @ttoine?

**Review by eternoendless — CHANGES_REQUESTED** [review]
Blocking this until a decision is made

**Comment by andromaque** [comment]
> After all, PrestaShop SA still largely funds PrestaShop development.

And there are also other contributors who spend a lot of time and efforts in this project, so they should be allowed to also advertise in the source code or documentation? I think it should remain neutral, advertise all you want in the PrestaShop corp site, on ads or whatever, but leave the project free of publicity.

**Comment by jf-viguier** [comment]
"are sold at reasonable prices"
"More than 3000 themes are available"
It's clearly a marketing stuff that doesn't have to be here in a dev doc.
I suggest that we will add "partner" marketplaces when foundation will be there and funded.

**Comment by jf-viguier** [comment]
"Blocking this until a decision is made" I will wait, no problemo.
Prestashop employees are not neutral either, maybe you should ask external maintainers @PululuK @kpodemski ?

**Comment by kpodemski** [comment]
@jf-viguier I'm a PrestaShop employee now.

**Comment by jf-viguier** [comment]
@kpodemski congrats 👍

**Comment by ghost** [comment]
I agree with @jf-viguier  we should put Prestashop 50 px and Okom3pom 3 px in the doc.

We have heard << PrestaShop (code) belongs to its authors >>, the same goes for the documentation.

For the moment PrestaShop SA contributes a lot but this is the purpose of the split

And I don't see the point of ads in the devdoc, in the user doc why not?

**Comment by eternoendless** [comment]
> there are also other contributors (...), so they should be allowed to also advertise in the source code or documentation?

> I don't see the point of ads in the devdoc the user doc why not?

I think it _could_ be kept as informational, using a less marketing-y wording.

The real question is: should we banish all mentions to marketplaces from the docs? Or should we list them all? Perhaps only those of partners who contribute significantly to the project? Or just those that finance the (future) foundation? All arguments are valid, and are being actively discussed in order to find the most suitable solution.

And who makes that call?

Currently, the PrestaShop company has the final word on the project's decisions – as it's always been since the project was born. The problem with that is that we can't expect people to appropriate the project if it's tightly controlled by a benevolent dictator. That's why we are working to redefine the project's governance (i.e. how decisions are made and by whom) by stating clear rules under which the PrestaShop company continues to influence the project, but just as one actor among many. 

The governance question here is under which circumstances other actors can be allowed to join the influence group (i.e. the "foundation"). Naturally, we can't just make it so that anyone that walks through the door automatically gains the right to vote – that wouldn't make any sense. My personal view is that you must _earn_ your vote either by occupying a specific role in the project (e.g. a maintainer), or by financing the foundation.

As I said, these rules haven't been defined yet, which is why I don't want to rush into decisions. It doesn't necessarily mean the content will stay exactly as it is: perhaps it will be removed, changed to refer to a sponsor marketplace, or perhaps just listed among all other known marketplaces.

Rest assured that we want to make things right.

**Comment by jf-viguier** [comment]
Thanks Pablo, I am confident, we are on the same track.
For now, I'll suggest another version for this pr.

**Comment by jf-viguier** [comment]
@eternoendless and others friends, I've posted a new version with a cleaner and more objective point of view on addons. IMO the technical validation is the main argument of addons on a dev point of view and should be enforced the same way in others potentially listed marketplaces.
I've introduced others marketplaces and removed marketing bullshit.
WDYT ? Do I deserve a goodie from addons marketing team ? 😉

**Comment by matks** [comment]
Thanks @jf-viguier but I think I heard some news that you will really love ;)

Waiting for @eternoendless confirmation but I think we got the GO to delete Addons stuff 🎉

**Review by eternoendless — APPROVED** [review]

**Comment by eternoendless** [comment]
Looks good to me. It looks like we're going to progressively delete content related to the marketplace.

**Review by matks — COMMENTED** [review]
@jf-viguier We got the OK to remove the Addons link. Do you want to revert your PR to the initial change before we merge? Or do you want to keep the latest version?

**Comment by jf-viguier** [comment]
Thanks @matks for this amazing decision, I've reverted and changed the last sentence, enjoy.

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thanks for the revert @jf-viguier and it's now approved and merged

**Pull request merged**
